compiler/ast.nim

```diff
@@ -1053,6 +1053,14 @@ const
   defaultAlignment = -1
   defaultOffset = -1
 
+template getStrVal*(a: PNode): string =
+  ## Returns underlying string for `{nkStrLit..nkTripleStrLit,nkSym,nkIdent}`.
+  let a2 = a
```

Comment:
Don't make it a template and use `a` directly.

Reply:
this avoids a copy; and lent is not possible.

Reply:
found a solution that avoids a copy (and uses a proc)